qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/template/TemplateFileTextDocumentService.java

```diff
@@ -84,14 +86,14 @@ public TemplateFileTextDocumentService(QuteLanguageServer quteLanguageServer, Sh
 	public void didOpen(DidOpenTextDocumentParams params) {
 		QuteTextDocument document = (QuteTextDocument) documents.onDidOpenTextDocument(params);
 		document.getProjectInfoFuture() //
-				.thenAccept(projectInfo -> {

```

Comment:
Could you please don't update the format of the code in this file. It is hard to review it. The main problem is that your formatter (from vscode-java) seems not have the same behavior than Eclipse IDE formatter.